beacon-chain/core/epoch/precompute/processing.go

```diff
@@ -0,0 +1,194 @@
+package precompute
+
+import (
+	"bytes"
+
+	"github.com/pkg/errors"
+	"github.com/prysmaticlabs/prysm/beacon-chain/core/helpers"
+	pb "github.com/prysmaticlabs/prysm/proto/beacon/p2p/v1"
+)
+
+// New gets called at the beginning of process epoch cycle to return
+// pre computed instances of validators attesting records and total
+// balances attested in an epoch.
+func New(state *pb.BeaconState) ([]*Validator, *Balance) {
+	vp := make([]*Validator, len(state.Validators))
+	bp := &Balance{}
+
+	currentEpoch := helpers.CurrentEpoch(state)
+	prevEpoch := helpers.PrevEpoch(state)
+
+	for i, v := range state.Validators {
+		// Was validator withdrawable or slashed
+		withdrawable := currentEpoch > v.WithdrawableEpoch
```

Comment:
should this be `>=`?

Reply:
Nice catch! Just checked the spec, it should be `>=`